automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/runtimes/RuntimeCreationDiskSpec.scala

```diff
@@ -219,19 +220,29 @@ class RuntimeCreationDiskSpec
           }
         })
         _ <- deleteRuntimeWithWait(googleProject, runtimeName, false)
+        _ <- LeonardoApiClient.patchDisk(googleProject, diskName, UpdateDiskRequest(Map.empty, newDiskSize))
+        _ <- IO.sleep(5 seconds)
 
         // Creating new runtime with existing disk should have test.txt file and user installed package
         runtimeWithData <- createRuntimeWithWait(googleProject, runtimeWithDataName, createRuntime2Request)
         clusterCopyWithData = ClusterCopy.fromGetRuntimeResponseCopy(runtimeWithData)
         _ <- IO(withWebDriver { implicit driver =>
-          withNewNotebook(clusterCopyWithData, Python3) { notebookPage =>
-            val persistedData =
-              """! cat /home/jupyter/test.txt""".stripMargin
-            notebookPage.executeCell(persistedData).get should include("this should save")
-            val persistedPackage = "! pip show beautifulSoup4"
-            notebookPage.executeCell(persistedPackage).get should include(
-              "/home/jupyter/.local/lib/python3.7/site-packages"
-            )
+          withNewNotebook(clusterCopyWithData, Python3) {
+            notebookPage =>
+              val persistedData =
+                """! cat /home/jupyter/test.txt""".stripMargin
+              notebookPage.executeCell(persistedData).get should include("this should save")
+              val persistedPackage = "! pip show beautifulSoup4"
+              notebookPage.executeCell(persistedPackage).get should include(
+                "/home/jupyter/.local/lib/python3.7/site-packages"
+              )
+
+              val res = notebookPage
+                .executeCell(
+                  "! df -H |grep sdb"
+                )
+                .get
+              res should include("158G")
```

Comment:
Nice.. thanks for the integration test.

For this check, I wouldn't be confident this value is stable, since some of the partition is not available after formatting. I imagine this could change slightly across GCE image updates... but it's possible it won't. Allowing some error margin here might be better (e.g. parse numeric value, allow +/- 5)

Reply:
If `158` is expected to grow, I wonder if it should be derived from `newDiskSize` so we don't have to update another place in the test.